{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/sqlalchemy_models/__init__.py

```diff
@@ -60,6 +87,9 @@ def includeme(config):
     settings['tm.manager_hook'] = 'pyramid_tm.explicit_manager'
 
     # use pyramid_tm to hook the transaction lifecycle to the request
+    # pyramid_tm and transaction will automatically close the database session
```

Comment:
These new comments need some punctuation and sentence capitalization to make intent more clear.

Also I think this line needs a "the" before "transaction".

Reply:
ok.

Reply:
"transaction" is the package name, like "pyramid_tm". What can I do to make this more clear?  under the hood, "pyramid_tm" doesn't actually close the active session, it relies on an implementation detail in the "transaction" library it utilizes, which is what actually calls "close()".